src/main/java/no/nav/pto/veilarbportefolje/tiltakshendelse/TiltakshendelseRepository.kt

```diff
@@ -65,6 +66,19 @@ class TiltakshendelseRepository(private val db: JdbcTemplate) {
         }
     }
 
+    fun slettTiltakshendelseOgHentEldste(tiltakshendelseId: UUID, fnr: Fnr): Tiltakshendelse? {
+        val sql = "DELETE FROM tiltakshendelse WHERE id = ?"
+        try {
+            db.update(sql, tiltakshendelseId)
+            return hentEldsteTiltakshendelse(fnr)
+        } catch (e: KunneIkkeHenteEldsteTiltakhendelseException) {
```

Comment:
Should we have some log here?

Reply:
it gets logged at lone 90 when the exception is thrown 👍